src/components/canvas/Effects.tsx

```diff
@@ -63,8 +67,8 @@ const MultiOutline = React.memo(
         selections2={selections2}
         selectionLayer={10}
         width={width}
-        edgeColor1={hColor as any}
```

Comment:
There is also

```jsx
import { ReactThreeFiber } from '@react-three/fiber'

ReactThreeFiber.Color
```

which corresponds to 

```tsx
export type Color = ConstructorParameters<typeof THREE.Color> | THREE.Color | number | string
```

which should be more accurate than THREE.ColorRepresentation which is just Color | string | number

overall, making all of these `any`, is that a good idea?